src/DataLoader/DataLoaderBuilder.cs

```diff
@@ -10,30 +10,33 @@ namespace Chinook.DataLoader
 	/// </summary>
 	public class DataLoaderBuilder : IDataLoaderBuilder
 	{
+		/// <summary>
+		/// Represents an empty <see cref="DataLoaderBuilder"/>.
+		/// </summary>
 		public static DataLoaderBuilder Empty => new DataLoaderBuilder();
 
-		/// <inheritdoc />
+		/// <inheritdoc cref="IDataLoaderBuilder.Name"/>
```

Comment:
Do you really have to specify the cref?

Reply:
We do not need to, but I find it more useful this way. Not sure why we would remove it completely.